src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/ScenarioHelpers.cs

```diff
@@ -69,8 +69,8 @@ public static async Task<int> RunScenarioAsync(Func<ILogger, Task<int>> func, Ca
             return result;
         }
 
-        public static async Task<int> RunWebScenarioAsync<TStartup>(Func<ILogger, Task<int>> func, CancellationToken token)
-            where TStartup : class
+        public static async Task<int> RunWebScenarioAsync<TStartup>(Action<IServiceCollection> configureServices, Func<ILogger, Task<int>> func, CancellationToken token)
```

Comment:
nit: Let's move the new parameter to the end and make it null by default so that the overload is not necessary.